cpp_ext/TorchOps.cpp

```diff
@@ -2,119 +2,95 @@
 // Created by maksim on 5/14/23.
 //
 
-#include "TorchOps.h"
+#include "Globals.h"
 #include "IRModule.h"
-#include "TorchTensor.h"
 
 #include <pybind11/pybind11.h>
 #include <pybind11/pytypes.h>
 #include <string>
 
-#include "Globals.h"
+#include "TorchOps.h"
+#include "TorchTensor.h"
 #include "TorchTypes.h"
 #include "TorchValues.h"
```

Comment:
@123epsilon @brucekimrokcmu just a heads up - there's a rough order convention here - upstream headers, then stl (and pybind) headers, then our headers.